Microsoft.HealthVault.Fhir.UnitTests/ToHealthVaultTests/AddressToHealthVaultTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Get Real Health.  All rights reserved.
+// MIT License
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the ""Software""), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+using System.Linq;
+using Hl7.Fhir.Model;
+using Microsoft.HealthVault.Fhir.Transformers;
+using Microsoft.HealthVault.ItemTypes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.HealthVault.Fhir.UnitTests.ToHealthVaultTests
+{
+    [TestClass]
+    [TestCategory("Fhir to ItemTypes")]
+    public class AddressToHealthVaultTests
+    {
+        [TestMethod]
+        public void WhenAddressTransformedToHealthVault_ThenValueEqual()
+        {
+            string[] lines = { "3804 Lakeland ", "Terrace" };
```

Comment:
Let's just use the Microsoft address here as opposed to something that could be real.
1 Microsoft Way
Redmond, Wa 98052

For district you can use King County. If you want a second line, you can use Studio B.